extensions/amp-accordion/1.0/accordion.js

```diff
@@ -189,6 +198,7 @@ export function AccordionSection({
 
   const expanded = isExpanded ? isExpanded(id, defaultExpanded) : expandedState;
   const animate = contextAnimate ?? defaultAnimate;
+  const contentId = `${prefix || 'a'}_AMP_content_${id}-${generateRandomId()}`;
```

Comment:
Doing a `generatorRandomId()` here inline is not great b/c it will change the value on each rerender. Probably won't break anything, but definitely extra work. I recommended above to just switch section `id` to `randomIdGenerator`. It's really not important how ID is generated, as long as it's unique in the scope of the accordion. If it ends up being globally unique - that's ok too.

Reply:
As a compromise, I've moved `generateRandomId` into a `useState` in the latest revision.  My rationale is that the sequential id generated by `generateSectionId` actually conveys information to the user about which section we are in (it's like an index).

Happy to change it though if you feel like this is overkill.

Reply:
I think it's not super necessary. But if it helps for debugging or conveys some information, it's definitely ok. Just please make sure to avoid drawing any kind of assumptions from ID ordering - we should not assume or rely on any ordering in our code.

Reply:
Got it, okay no assumptions will be made based on ID ordering.